source/release-notes.txt

```diff
@@ -10,6 +10,58 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.38.0
+----------------
+
+*Released June 21, 2023*
+
+New Features:
+
+- OIDC authentication (:issue:`COMPASS-6803`).
+- Stage wizard (:issue:`COMPASS-6814`).
+- Add visual tree and update summary for the aggregation explain plan
+  (:issue:`COMPASS-6821` and :issue:`COMPASS-6888`).
+- Add types docs link to import preview (:issue:`COMPASS-6632`).
```

Comment:
(question)
This seems fairly internal and probably safe to delete. Would you please check during the external review?

Reply:
Yup!

Reply:
Done.